src/data/items.txt

```diff
@@ -11140,3 +11140,23 @@
 11112	pixel bread	950236935	pixelbread.gif	food	t,d	30
 11113	pixel whiskey	262338395	pixelwhiskey.gif	drink	t,d	30
 11114	pixel rock	827961712	pixelrock.gif	grow	d	16384
+11115	S.I.T. Course Voucher	362646202	sitvoucher.gif	usable	t	0
+11116	S.I.T. Course Completion Certificate	495156664	sitcert.gif	usable		0
```

Comment:
you use it to get a choice encounter where you can pick from 3 choices, each of which gives you one of the skills for the day.

At the moment Mafia NPEs after picking a choice because it doesn't recognise the skill you get:

```
java.lang.NullPointerException: Cannot invoke "net.sourceforge.kolmafia.request.UseSkillRequest.getSkillName()" because "skill" is null
	at net.sourceforge.kolmafia.session.ResponseTextParser.learnSkill(ResponseTextParser.java:853)
	at net.sourceforge.kolmafia.session.ResponseTextParser.learnSkillFromResponse(ResponseTextParser.java:757)
	at net.sourceforge.kolmafia.session.ResponseTextParser.learnSkill(ResponseTextParser.java:740)
	at net.sourceforge.kolmafia.request.GenericRequest.processResults(GenericRequest.java:2380)
```

Reply:
I have a branch to fix the NPE, but I'll need to test it tomorrow, because you can only use the choice once per day.